sympy/algebras/quaternion.py

```diff
@@ -303,6 +303,8 @@ def normalize(self):
     def inverse(self):
         """Returns the inverse of the quaternion."""
         q = self
+        if not q.norm():
+            raise ValueError("Cannot compute inverse for Quaternion(0,0,0,0)")
```

Comment:
Perhaps something like "Cannot compute inverse for a quaternion with zero norm".

Reply:
That would also be a good.